app/scripts/modules/netflix/pagerDuty/pagerDutyTag.component.spec.ts

```diff
@@ -0,0 +1,43 @@
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+
+import {PagerDutyTagComponent} from './pagerDutyTag.component';
+import {PagerDutyReader} from 'netflix/pagerDuty/pagerDuty.read.service';
+
+describe('PagerDutyTagComponent', () => {
+
+  let component: PagerDutyTagComponent;
+  let fixture: ComponentFixture<PagerDutyTagComponent>;
+  // let element: HTMLElement;
+
+  const pagerDutyReaderStub = {
+    listServices: () => {
+      return [
+        {name: 'name', integration_key: 'integrationKey'}
+      ];
+    }
+  };
+
+  beforeEach(() => {
+    // module('ANGULAR_INJECTOR_MODULE', []).factory('$$angularInjector', ['$injector', function ($injector: ng.auto.IInjectorService) {
+    //   TestBed.configureTestingModule({
+    //     declarations: [PagerDutyTagComponent],
+    //     providers: [
+    //       {provide: '$injector', useValue: $injector},
+    //       {provide: PagerDutyReader, useValue: pagerDutyReaderStub}
+    //     ]
+    //   });
+    //   return TestBed.get(Injector);
+    // }]);
```

Comment:
Since we're not adding any new functionality, it would be really nice to hold off merging this until we can figure out how to get the tests running.